Update active-directory-saas-jostle-tutorial.md

JostleVince opened the pull request:
Edited content and screenshots to reflect the updated workflow of this process.

azurecla commented:
Hi __@DevXE__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@DevXE : Thanks for your contribution to the Azure documentation! The author, @jimdial, has been notified to review your proposed change.

jimdial commented:
Thanks for catching this inconsistency with the limits article @DevXE. I proposed a change to your change (with a comment explaining why). If you're OK with my change, please merge it and I can get your PR merged.  Thanks.

DevXE commented:
Hi @jimdial - I'm seeing any changes you've proposed.

jimdial commented:
@DevXE : Hmmm, don't know what happened, but I've proposed the changes to your PR [here](https://github.com/DevXE/azure-content/pull/1). If you're good with them, just merge the request and I'll merge yours.  --Thanks

jimdial commented:
Thanks again for the feedback @DevXE. #sign-off

JostleVince committed:
Update active-directory-saas-jostle-tutorial.md

Edited content and screenshots to reflect the updated workflow of this process.
